grails-app/domain/mama/ng/scheduler/Message.groovy

```diff
@@ -0,0 +1,16 @@
+package mama.ng.scheduler
+
+class Message {
+
+    static hasOne = [schedule: Schedule]
+
+    String id = UUID.randomUUID().toString()
+    Date nextSend
+    Date dateCreated
```

Comment:
dateCreated is a special character that grails fills in for you ie it will always be new Date() when the object is created.